index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="css/style.css">
+  <link
+  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+  rel="stylesheet"
+/>
+</head>
+<body>
+  <header>
+      <nav>
+          <div class="logo-part">
+              <h4 class="logo-text">My Logo</h4>
+          </div>
+          
+
+              <ul class="hamburgur">
+                      
+                      <span class="bar"></span>
+                      <span class="bar"></span>
+                      <span class="bar"></span>
+              </ul>
+
+          </ul>
+      </nav>
+  </header>
+
+  <section class="main">
+      <div class="background">
+          <div class="wrapper">
+              <div class="intro-text">
+                  <h1>
+                      I'm CarenSiya
+                      <br>
+                      Glad to see you!
+                  </h1>
+                  <p></p>
+              </div>
+              <div class="main-text">
+                  <p class="text-description">
+                      I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                  </p>
+                  <div class="social">
+                      <h2 class="connection">LET'S CONNECT</h2>
+                      <ul class="media-icons">
+                          <li>
+                              
+                                  <i class="fa-brands fa-twitter"></i>
+                              </a>
```

Comment:
- [ ] [ **OPTIONAL** ] I see the closing tag here but there is no opening tag for `<a>`. This might create errors in the future. Please fix this for the rest of social icons to avoid any unknown errors 👍🏽 🙂